src/lib/provable/test/test-utils.ts

```diff
@@ -62,6 +66,26 @@ function bytes(length: number) {
   });
 }
 
+function pointSpec<T>(field: ProvableSpec<bigint, T>, Curve: CurveAffine) {
```

Comment:
just a testing utility, ends up unused (I used it during development to monitor the size of some circuits)